services-directions/src/test/java/com/mapbox/api/directions/v5/models/RouteLegTest.java

```diff
@@ -84,7 +86,7 @@ public void testToFromJson1() {
       .distance(53.4)
       //.weight(14.3)
       .duration(14.3)
-      .steps(new ArrayList<>())
+      .steps(legSteps)
```

Comment:
these changes seem unrelated but no biggy to keep them around

Reply:
For some reason it was giving me compilation error under java 7.
Casting was not possible for some reason.